tests/legacy-cli/e2e/tests/build/lazy-load-syntax.ts

```diff
@@ -27,32 +29,44 @@ export default async function () {
       exports: [RouterModule]
     })
     export class AppRoutingModule { }
-  `);
-  await prependToFile('src/app/app.module.ts',
-    `import { AppRoutingModule } from './app-routing.module';`);
+  `,
+  );
```

Comment:
Nit: If the formatter allows it, I recommended lining up indents between the starting and ending backticks (and maybe indent the text one more level), which can help readers associate the start and end tokens together.

```typescript
await writeFile(
  appRoutingModulePath,
  `
    import { NgModule } from '@angular/core';
    // ...
  `,
);
```